src/MOCK_DB_DATA/library-items/library-items-shared-data.ts

```diff
@@ -181,9 +173,17 @@ export const LibraryItemsSharedData: LibraryItemsShared = {
     id: 'time-selector-week-range',
     title: 'Week range',
   },
-  TimeSelectorYear: {
-    id: 'time-selector-year',
-    title: 'Year selector',
+  TimeSelectorYearMonth: {
+    id: 'time-selector-year-month',
+    title: 'Year month',
+  },
+  TimeSelectorYearMonthRange: {
+    id: 'time-selector-month-range',
+    title: 'Month range',
+  },
+  TimeSelectorYears: {
+    id: 'time-selector-years',
+    title: 'Years',
```

Comment:
Year, singular? Or will this be a multiselect eventually?

Reply:
Yes, will become a multiselect